socketcan_bridge/include/socketcan_bridge/topic_to_socketcan.h

```diff
@@ -50,19 +50,7 @@ class TopicToSocketCAN
     void stateCallback(const can::State & s);
 };
 
-void convertMessageToSocketCAN(const can_msgs::Frame& m, can::Frame& f)
-{
-  f.id = m.id;
-  f.dlc = m.dlc;
-  f.is_error = m.is_error;
-  f.is_rtr = m.is_rtr;
-  f.is_extended = m.is_extended;
-
-  for (int i = 0; i < 8; i++)  // always copy all data, regardless of dlc.
-  {
-    f.data[i] = m.data[i];
-  }
-};
+void convertMessageToSocketCAN(const can_msgs::Frame& m, can::Frame& f);
```

Comment:
It seems the only changes here are that the definition is moved to the source file. Is that necessary or only to comply with our standards? As otherwise the files stay untouched I would prefer to not change them if not necessary.

Reply:
I think there is a problem for the definition stay in the header. If I remember correctly, if I include the same header from more than one .cpp file, then the function would have duplicated definition and cause linker error.

Reply:
Okay, I see, then it's okay I suppose